src/persist-client/src/impl/state.rs

```diff
@@ -35,6 +37,12 @@ pub struct ReadCapability<T> {
     pub since: Antichain<T>,
 }
 
+#[derive(Clone, Debug, PartialEq)]
+pub struct WriterState {
+    /// UNIX_EPOCH timestamp of the last time this writer recorded a heartbeat
+    pub last_heartbeat: u64,
```

Comment:
note that this is in millis in the comment (or alternatively maybe include it in the name?)